packages/toolpad-app/src/appDom/index.ts

```diff
@@ -235,7 +237,7 @@ function assertIsType<T extends AppDomNode>(node: AppDomNode, type: T['type']):
 }
 
 function createId(): NodeId {
```

Comment:
```tsx
import { init } from '@paralleldrive/cuid2';

const createIdInternal = init({ length: 7 })

export function createId(): NodeId {
  return createIdInternal() as NodeId;
}

Reply:
This still fails on `yarn test` with the same error as in the CI

Reply:
yes, I wasn't proposing this as a fix for that, just as a suggestion to use the lib in a more idiomatic way.

Let's just move to `nanoid/non-secure` instead of `cuid2`. That's more than suitable for our needs and won't have these issues